### PR TITLE
Feature/header banner

### PR DESCRIPTION
**Pull request by Matveeva-Ann**

### CODE REVIEW

File: css/banner.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
+.banner {
+  display: flex;
+  align-items: center;
+}
+
+@media (width < 650px) {
```
Comment (YuriiSkyba1) [inline css/banner.css:6]: usually we move all media to bottom on the file

File: css/banner.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  line-height: clamp(30px, 3.3vw, 55px);
+  margin-bottom: 18px;
+}
+
+.banner__title--accent {
+  color: var(--accent-color);
+}
+
+.banner__text {
+  font-size: clamp(var(--font-size-small), 1vw, var(--font-size-base));
```
Comment (YuriiSkyba1) [inline css/banner.css:48]: nice approach

File: index.html [hunk 3]
```diff
@@ -1,15 +1,81 @@
...
+            <button class="banner__bunDiscover">Discover Now</button>
+            <button  class="banner__bunDemo">Watch Demo</button>
+          </div>
+        </div>
+        <img class="banner__img" src="./images/Bunner.jpg" alt="bunner" />
+      </section>
+    </div>
+    <section class="brands container">
+      <ul class="brands__list">
+        <li><img src="./images/Group.jpg" alt=""></li>
```
Comment (YuriiSkyba1) [inline index.html:72]: missed alt

File: index.html [hunk 4]
```diff
@@ -1,15 +1,81 @@
...
+        <li><img src="./images/Group (1).jpg" alt=""></li>
+        <li><img src="./images/Group (2).jpg" alt=""></li>
```
Comment (YuriiSkyba1) [inline index.html:74]: wrong naming for images, should be without numbers

**Review by YuriiSkyba1 — APPROVED** [review]

**Comment by Matveeva-Ann** [comment]
I fixed bugs from the previous pull request in this pull request. Moving forward, I will fix bugs in the pull request where they were originally made to avoid mixing different tasks in one pull request

**Pull request merged**